index.js

```diff
@@ -1,10 +1,8 @@
 'use strict';
+const {prototype, getPrototypeOf} = Object;
+const {toString} = prototype;
```

Comment:
I don't really see the point of destructuring here. V8 is smart enough to optimize property access away, so this is just verbosity for no benefit.